doc/STYLE_GUIDE.md

```diff
@@ -29,8 +29,8 @@
   `[a link](http://example.com)`.
 * When documenting APIs, update the YAML comment associated with the API as
   appropriate. This is especially true when introducing or deprecating an API.
-* Use [Em dashes][] ("—" or `Option+Shift+"-"` on macOS) surrounded by spaces,
-  as per [The New York Times Manual of Style and Usage][].
+* Use [Em dashes][] ("—" or `Option+Shift+"-"` on macOS, `Shift+Ctrl+U 2014 Enter` on many Linux systems)
+  surrounded by spaces, as per [The New York Times Manual of Style and Usage][].
```

Comment:
Fwiw, AltGr + Shift + - works for me, and is a bit more intuitive/similar to the Mac one

Reply:
Here it produces the ˙ modifier for letters like ṁ, but if it works on other keyboards, I will add it!

Reply:
The style guide changes are unrelated and probably shouldn't be in this PR?

(Also, I wouldn't mind changing the style guide to tell people to avoid dashes, to be honest. They tend to get used where they aren't required, and there are too many typographical variations--em dash vs. en dash vs. spaces around the dash vs. no space around the dash ve.....)

Reply:
Yes, removed the changes again from the PR.